.github/workflows/build.yml

```diff
@@ -11,11 +11,11 @@ defaults:
     shell: bash -l {0}
 
 jobs:
-  build:
+  lint:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@v2
```

Comment:
This downgrade to `v2` is probably unwanted?

Reply:
Thanks for the review, indeed that's a rebase issue